main.go

```diff
@@ -98,7 +100,9 @@ func main() {
 		panic(fmt.Errorf("can't use both schema-only and skip-foreign-keys at once. Foreign Key creation can only be skipped when data migration takes place."))
 	}
 
-	ioHelper := &conversion.IOStreams{In: os.Stdin, Out: os.Stdout}
+	input := loadInput(dumpFile)
```

Comment:
nit; should we add some defensive code to check that dumpFile is passed in only in the case of driver being mysqldump or pgdump?

Reply:
Based on the current handling of the other flags i think this is fine. Adding conditional code here for a specific case seems a bit odd to me.

I think we should handle this as a part of the SubCommands change you are taking up, wherein we should take a call on whether we should ignore parameters that are not relevant to the current flow or to raise an error.

Reply:
Sounds reasonable, I agree that adding more robust handling as part of subcommands is the way to go forward here.